lib/routes/home/widgets/bottom_actions_bar/send_options_bottom_sheet.dart

```diff
@@ -47,9 +44,9 @@ class SendOptionsBottomSheet extends StatelessWidget {
             ),
             ListTile(
               enabled: false, // TODO: back to connected when we integrate with the SDK
-              leading: BottomActionItemImage(
+              leading: const BottomActionItemImage(
                 iconAssetPath: "src/icon/bitcoin.png",
-                enabled: connected,
+                enabled: false, //  TODO: back to connected when we integrate with the SDK
```

Comment:
Why is that?

Reply:
I have now updated the comment to avoid confusion.
 
This was an earlier comment dependent on whenever we where connected to the LSP, where we disabled the button when not connected to the LSP.